components/date-picker/index.zh-CN.md

```diff
@@ -114,6 +114,7 @@ moment.locale('zh-cn');
 | format | 展示的日期格式，配置参考 [moment.js](http://momentjs.com/) | string | "YYYY-wo" |
 | value | 日期 | [moment](http://momentjs.com/) | - |
 | onChange | 时间发生变化的回调，发生在用户选择时间时 | function(date: moment, dateString: string) | - |
+| renderExtraFooter | 在面板中添加额外的页脚 | (mode) => React.ReactNode | - |
```

Comment:
You can help me remove `renderFooter` I just added above by mistake... 😆

Reply:
https://github.com/ant-design/ant-design/commit/3c007a85dd8a670f66c7e0aed95d3537e25ef6ea

Reply:
fixed